src/app/modules/user-manage/user-manage-list/user-manage-list.component.html

```diff
@@ -0,0 +1,119 @@
+<div class="settings-bar">
+  <mat-form-field appearance="standard">
+    <mat-label>{{ "Filter" }}</mat-label>
+    <input
+      matInput
+      (keyup)="applyFilter($event)"
+      placeholder="Ex. ium"
+      #input
+    />
+  </mat-form-field>
+  <span class="spacer"></span>
+  <button mat-stroked-button [disabled]="!selection.selected.length">
+    {{ "Block" }}
+  </button>
+  <button mat-stroked-button [disabled]="!selection.selected.length">
+    {{ "Unblock" }}
+  </button>
+  <button
+    mat-stroked-button
+    [disabled]="!selection.selected.length"
+    color="warn"
+  >
+    {{ "Delete" }}
+  </button>
+  <span class="spacer"></span>
+  <mat-form-field appearance="standard">
+    <mat-label>{{ "Select user status" }}</mat-label>
+    <select matNativeControl (change)="userStatusChanged($event)">
+      <option
+        *ngFor="let status of userStatuses"
+        [value]="status.value"
+        [selected]="selectedUserStatus === status.value"
+      >
+        {{ status.viewValue }}
+      </option>
+    </select>
+  </mat-form-field>
+</div>
+
+<table mat-table [dataSource]="usersList" matSort class="mat-elevation-z8">
+  <!-- Checkbox Column -->
+  <ng-container matColumnDef="select">
+    <th mat-header-cell *matHeaderCellDef>
+      <mat-checkbox
+        (change)="$event ? toggleAllRows() : null"
```

Comment:
`[2]` do not load templates with logic. All logic should be in the component class, templates are only responsible for the display